src/cluster_info.rs

```diff
@@ -804,34 +804,38 @@ impl ClusterInfo {
             })
             .unwrap()
     }
+
+    // TODO: To support repairing multiple slots, change broadcast to support
```

Comment:
is this comment still correct?

Reply:
I believe so, repair requests don't include the slot index, and broadcast needs to switch to a per-slot blob indexes instead of a global blob index.

Reply:
is this a long pole/hard thing?

Reply:
I think the broadcast part should be fine, it's just resetting the blob index when we update the slot index. I think the more critical part will be https://github.com/solana-labs/solana/issues/2444

Reply:
actually, @aeyakovenko and I think the current send() interface with block/slot info will be sufficient, no need for a pull-oriented API from the ledger